conf.c

```diff
@@ -1143,22 +1143,22 @@ void conf_brcfg(FILE *output, int ifs, struct if_nameindex *ifn_list,
 
 	if ((l_tmp = bridge_cfg(ifs, ifname, PRIORITY))
 	    != -1 && l_tmp != DEFAULT_PRIORITY)
-		fprintf(output, " priority %lu\n", l_tmp);
```

Comment:
This whole chunk of changes is identical to pull request 112 which has been merged.

Reply:
Yeah it should be fine. If there was an actual merge conflict I believe it would already be shown in github's UI at this point.